### PR TITLE
[REACTOS] "http://www.reactos" -> "https://reactos"

### PR DESCRIPTION
**Pull request by JoachimHenze**
to harmonize,
to save some bytes,
and last but not least for security reasons.

This can be understood as an addendum to (#2619).

### CODE REVIEW

File: base/setup/usetup/lang/ru-RU.h [hunk 1]
```diff
@@ -161,7 +161,7 @@ static MUI_ENTRY ruRUWelcomePageEntries[] =
     {
         6,
         24,
-        "http://www.reactos.ru",
+        "https://reactos.ru",
```
Comment (GGLinnk) [inline base/setup/usetup/lang/ru-RU.h:164]:
```suggestion
        "https://reactos.org",
```

[http~~s~~://reactos.ru](https://reactos.ru/) redirects to https://reactos.org but with ssl issues.

File: base/setup/usetup/lang/uk-UA.h [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // To revert conversion, please execute "code7bit -r <file>".
 /*
  *      translated by Artem Reznikov, Igor Paliychuk, 2010
- *      http://www.reactos.org/uk/
+ *      https://reactos.org/uk/
```
Comment (ColinFinck) [inline base/setup/usetup/lang/uk-UA.h:6]: Broken URL

**Review by ColinFinck — CHANGES_REQUESTED** [review]
See comments

File: base/setup/usetup/lang/ru-RU.h (hunk 1 above)
Comment (sashaqwert) [reply base/setup/usetup/lang/ru-RU.h:164]: Also exists https://reactos.org/ru redirect

Comment (JoachimHenze) [reply base/setup/usetup/lang/ru-RU.h]: fixed as proposed by 2upvoted GGLink proposal.

File: base/setup/usetup/lang/uk-UA.h (hunk 2 above)
Comment (JoachimHenze) [reply base/setup/usetup/lang/uk-UA.h:6]: fixed

Comment (GGLinnk) [reply base/setup/usetup/lang/uk-UA.h:6]: Still broken in comment

```suggestion
 *      https://reactos.org/
```

**Review by GGLinnk — CHANGES_REQUESTED** [review]

File: base/setup/usetup/lang/uk-UA.h (hunk 2 above)
Comment (JoachimHenze) [reply base/setup/usetup/lang/uk-UA.h]: I fixed that 2nd one as well.

**Review by ColinFinck — APPROVED** [review]

File: drivers/storage/ide/uniata/id_init.cpp [hunk 3]
```diff
@@ -2555,7 +2555,7 @@ AtapiChipInit(
                 * size goes below 32DW.  Setting it to 1 makes the watermark
                 * 64DW.
                 *
-                * http://www.reactos.org/bugzilla/show_bug.cgi?id=6500
+                * https://reactos.org/bugzilla/show_bug.cgi?id=6500
```
Comment (GGLinnk) [inline drivers/storage/ide/uniata/id_init.cpp:2558]: Broken link.
Seems to be https://jira.reactos.org/browse/CORE-5897

```suggestion
                * https://reactos.org/bugzilla/show_bug.cgi?id=6500
                * https://jira.reactos.org/browse/CORE-5897
```

Comment (JoachimHenze) [reply drivers/storage/ide/uniata/id_init.cpp:2558]: I do hope so, because we are beginning to become quite offtopic. :-)

**Review by GGLinnk — APPROVED** [review]

**Comment by HBelusca** [comment]
Notice that if it were just "for security reasons", using "https" in the www.reactos.org URL would have been enough.

**Pull request merged**